setup.py

```diff
@@ -22,10 +22,7 @@
         'pytest>=2.9,<3.0',
         'six',
     ],
-    packages=[
-        'dynamallow',
-    ],
-
+    packages=find_packages('.', exclude=['tests']),
```

Comment:
Updated. calling find_packages with `dynamallow` did not work, so I had to resort to this.